test/common/stats/symbol_table_impl_test.cc

```diff
@@ -732,11 +732,13 @@ TEST(SymbolTableTest, Memory) {
     }
   }
 
-  // Make sure we don't regress. Data as of 2019/05/29:
-  //
+  // Make sure we don't regress.
+  // Data as of 2019/05/29:
   // string_mem_used:        6710912 (libc++), 7759488 (libstdc++).
+  // Data as of 2020/09/23 (switch to https://github.com/google/tcmalloc):
+  // string_mem_used:        6710912 (libc++), 7775872 (libstdc++).
   // symbol_table_mem_used:  1726056 (3.9x) -- does not seem to depend on STL sizes.
-  EXPECT_MEMORY_LE(string_mem_used, 7759488);
+  EXPECT_MEMORY_LE(string_mem_used, 7775872);
```

Comment:
CI fails on gcc now with:

Expected: (string_mem_used) <= (7775872), actual: 7792256 vs 7775872

I'm guessing this may be related to varying STL versions for std::string across platforms than tcmalloc. I think we can just remove the exact check for string_mem_used. We can keep the test that symbol tables cut the usage by 2/3.